### PR TITLE
Add chocolatey install instructions

### PR DESCRIPTION
**Pull request by lukebakken**
https://github.com/chocolatey-beam/elixir-package

### CODE REVIEW

File: install.markdown [hunk 1]
```diff
@@ -96,6 +96,9 @@ The preferred option for installing Elixir. Choose your operating system and too
     * Install Erlang: `scoop install erlang`
     * Install Elixir: `scoop install elixir`
 
+  - Using Chocolatey:
```
Comment (josevalim) [inline install.markdown:99]:
```suggestion
  - Using [Chocolatey](https://community.chocolatey.org/):
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by lukebakken** [comment]
Fastest turnaround ever! My eventual goal is to get the chocolatey packages 100% auto-updating. Baby steps.

**Comment by josevalim** [comment]
That's awesome and it will be very welcome! There is a discussion about that here: https://github.com/erlef/build-and-packaging-wg/issues/42

**Comment by lukebakken** [comment]
Thanks for pointing that out. I would never have found that discussion.

**Pull request merged**